src/client/components/Editor/FlaggedPhrase.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+// App Imports
+import { phraseType } from '../../types'
```

Comment:
We should look at [module-alias](https://github.com/ilearnio/module-alias).

Reply:
Great idea -- I'll open an issue.